pkg/cmd/extension/manager.go

```diff
@@ -172,7 +179,80 @@ func (m *Manager) InstallLocal(dir string) error {
 	return makeSymlink(dir, targetLink)
 }
 
-func (m *Manager) Install(cloneURL string, stdout, stderr io.Writer) error {
+type BinManifest struct {
```

Comment:
Seems like keeping the version in the manifest will be important for checking for updates.